jd4/case.py

```diff
@@ -25,6 +25,7 @@
 DEFAULT_TIME_NS = 1000000000
 DEFAULT_MEMORY_BYTES = 268435456
 PROCESS_LIMIT = 64
+re_input = re.compile(r'([a-zA-Z]*)([0-9]*)\.in')
```

Comment:
Why `*` rather than `+`?

Reply:
Submit answer problems may only have one test case.
They can use 
data.in
data.out

Reply:
int('') will throw an error.
TODO: fix

Reply:
## :( Hard to fix.
Removing this feature.

Reply:
`0 if not match.group(2) else int(match.group(2))` ?

Reply:
Then uses give
```
data.in
data0.in
data1.in
```
...

Reply:
Then `-1` instead of `0`? (fog

Reply:
An error "data-1.in not found" occurs.

Reply:
??? `-1` would not match.

Reply:
Match what?
![image](https://user-images.githubusercontent.com/29992205/56091142-711cb180-5edd-11e9-8e88-45deaf9a32a8.png)